flang/docs/FlangDriver.md

```diff
@@ -339,11 +335,11 @@ just added using your new frontend option.
 
 ## CMake Support
 As of [#7246](https://gitlab.kitware.com/cmake/cmake/-/merge_requests/7246)
-(and soon to be released CMake 3.24.0), `cmake` can detect `flang-new` as a
+(and soon to be released CMake 3.24.0), `cmake` can detect `flang` as a
```

Comment:
```suggestion
(and CMake 3.24.0), `cmake` can detect `flang` as a
```
Can CMake already detect `flang` as well?

Reply:
This is a good question.  Also, can it detect our flang instead of classic Flang regardless of executable name, or is the detection of flang vs. flang-new the way CMake differentiates the two compilers?

Reply:
That I am not quite sure of. I will look into it and adjust that comment in the docs as appropriate.

Reply:
@banach-space appears to have been involved in that MR, do you happen to know the answer to the above questions? It looks like it doesn't rely on the `flang-new` name, but I'm not sure.

Reply:
1. A search in the CMake repository revel that `flang-new` only appear in testcases and ci-related code - not in any code that eventually end up in CMake. This indicate that the name `flang-new` is not a magic token for CMake to recognize LLVM Flang.
2. I think that [this is the merge](https://gitlab.kitware.com/cmake/cmake/-/merge_requests/7246) that first introduced LLVM Flang to CMake. It seems like the check is through preprocessor macros defined by the different compilers. See [`Modules/CMakeFortranCompilerId.F.in`](https://gitlab.kitware.com/cmake/cmake/-/blob/master/Modules/CMakeFortranCompilerId.F.in) in the CMake sources.

Reply:
>  It seems like the check is through preprocessor macros defined by the different compilers.

Yes, thank you @hakostra 🙏🏻

Reply:
Just to add, for an actually working CMake with flang-new that detects the correct options and does the right thing reliably, you'll need CMake 3.28 as that's the first version with fully working flang-new support, so if updating this comment could you change 3.24 to 3.28 as well? 
I am fairly sure the CMake support for LLVM flang is designed to work fine regardless of the name of the binary, as the CMake devs were anticipating this name change anyway.

Reply:
So, sounds like support is there in CMake, so I've updated the note and the identified version.